src/main/java/com/google/sps/servlets/InsertRepDatastoreServlet.java

```diff
@@ -58,12 +58,17 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
             throw new ServletException("Error: " + e.getMessage(), e);
         }
         if (rep == null) {
-            //Initializes rep's tab list with an Other tab prepended by repName for uniqueness
-            List<Long> tabIds = DatastoreManager.insertTabsInDatastore(
-                //Prepend the Other tag with repName for uniqueness and 
-                // remove spaces in repName with replaceAll
-                Arrays.asList(repName.replaceAll("\\s+","") + "Other"), 
-                Arrays.asList(OTHER_TAB_PLATFORM));
+            List<Long> tabIds;
+            //Initializes rep's tab list with an Other tab prepended by repName for 
+            //uniqueness and remove spaces in repName with replaceAll
+            try {
+                tabIds = DatastoreManager.insertTabsInDatastore( 
+                    Arrays.asList(repName.replaceAll("\\s+","") + "Other"), 
```

Comment:
keep the comment?

Reply:
I just condensed them cause it was getting repetitive